test/.eslintrc.js

```diff
@@ -5,6 +5,11 @@
 
 'use strict'
 module.exports = {
+  plugins: ['disable'],
```

Comment:
I pared this down because .eslintrc inherit from parents so you really only have to load or override what is different

Reply:
Thanks, I appreciate it!